src/playbooks/with_build/vstat_destroy.yml

```diff
@@ -9,7 +9,7 @@
       vars:
         destroy_components_name: VSTAT
 
-- hosts: vstats
+- hosts: vstats,data_vstats
```

Comment:
No, because the "vsds" group contains all of the VSDs including the stats_only ones.  The vstats group keeps them separate.  I don't like the inconsistency, but this is how it existed before with the standby systems.  Changing it risks breaking GEO redundancy which I was careful not to modify too much.  Note, I did a regression test for GEO and it still passed.